ludwig/datasets/sst2/__init__.py

```diff
@@ -14,19 +14,16 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 # ==============================================================================
-from ludwig.datasets.base_dataset import BaseDataset, DEFAULT_CACHE_LOCATION
-from ludwig.datasets.mixins.download import ZipDownloadMixin
-from ludwig.datasets.mixins.load import CSVLoadMixin
-from ludwig.datasets.mixins.process import *
+from ludwig.datasets.base_dataset import DEFAULT_CACHE_LOCATION
+from ludwig.datasets.sst2.sst_utils import SST
```

Comment:
what is SST, should we rename to something more descriptive :)

Reply:
It's the acronym for Stanford Sentiment Treebank, which is a bit long... :)